README.md

```diff
@@ -75,6 +80,38 @@ end
 
 ```
 
+```ruby
+Register Listening Tentacle
+# You will first need to generate an api key
+# In Octopus Deploy Server GUI click your Name -> Profile -> API keys
+octopus_deploy_tentacle 'Tentacle' do
+  action :register
+  server 'https://octopus.example.com'
+  api_key '12345678910'
+  roles ['database']
+  environment 'prod'
+  polling false
```

Comment:
Which can be left out? The role and environment? The server and api key are required for the idempotency check.